src/feature/user-profile/api/update-my-customer.ts

```diff
@@ -49,11 +52,12 @@ export async function changeCustomerPassword(
   version: number,
   currentPassword: string,
   newPassword: string,
-  token: string,
+  email: string,
 ): Promise<Customer> {
+  const token = tokenCache.get().token;
   const apiRoot = createApiClientWithToken(token);
```

Comment:
Токена тут не хватает

```suggestion
  email: string,
  token: string,
): Promise<Customer> {
  if (!token) throw new Error('Missing token');
  const apiRoot = createApiClientWithToken(token);
```